add-on/chrome/bootstrap.js

```diff
@@ -550,7 +569,11 @@ var WindowListener = {
 
         this._hideBrowserSharingInfoBar();
         gBrowser.tabContainer.removeEventListener("TabSelect", this);
-        gBrowser.removeEventListener("DOMTitleChanged", this);
+
+        if (titleChangedListener) {
+          this.mm.removeMessageListener("loop@mozilla.org:DOMTitleChanged",
+            titleChangedListener);
```

Comment:
Please do `titleChangedListener = null;`, otherwise this whole exercise is for naught...